manifests/gnome/gconftool_2.pp

```diff
@@ -39,9 +39,17 @@
     $type_real = $type
   }
 
+  if $::lsbmajdistrelease == '10' {
```

Comment:
When using $::osfamily in the code you'll need to add it to the mandatory_global_facts:
https://github.com/Phil-Friderici/puppet-module-gnomish/blob/master/spec/spec_helper.rb

That should remove a lot of failures when running the tests.